packages/css-framework/src/components/_dropdown.scss

```diff
@@ -1,124 +1,135 @@
+$borderThick: 2px;
```

Comment:
Elsewhere the convention seems to be hyphenated variable names (`$border-thick`).